overview/roles.md

```diff
@@ -91,81 +118,71 @@ typeset a sign. There are, however, incorrect ways that are not visually
-episode before it is released. They are responsible inspecting the
-overall quality of every aspect of a show including, timing,
-typesetting, encode, and edit. Quality Checkers are expected to be well
+Quality Checkers (abbreviated *QC*) are often the last eyes on an
+episode before it is released. They are responsible for inspecting the
+overall quality of every aspect of a show including: timing,
+typesetting, encode, and editing. Quality Checkers are expected to be well
 versed in every role and able to critique them. Each group has a
 different approach to how the Quality Checker completes their work. For
 example, one group might require an organized "QC report" of recommended
 changes and required fixes, while other groups may prefer the Quality
 Checker to make changes directly to the script, if they can.
 
-### Translator & Translation Checker
+## Translator & Translation Checker
 
-Time commitment per episode: 1-3 hours for TLC, 4+ hours for an original
-translation (dependent on the skill of the TL/C and the difficulty of
-the show)
+Time commitment per episode: 1-3 hours for translation check,
+4+ hours for an original translation (dependent on the skill of the TL/TLC
+and the difficulty of the show's original script)
 
 The job of the Translator (abbreviated *TL*) and the Translation Checker
 (abbreviated *TLC*) is to translate and ensure the translational quality
-of the script and translated signs. This is perhaps an obvious
-statement, but it bears explaining just in case. Today, many shows are
+of the script and signs respectively. This is perhaps an obvious statement,
+but it bears explaining just in case. Today, many shows are
 *simulcast* by one or more companies, meaning scripts will be available
-immediately after or soon after airing in Japan. In these cases, some
-fansub groups may choose to edit and translation check the simulcast
-script rather than translate it from scratch. This depends almost
+immediately or soon after airing in Japan. In these cases, some
+fansub groups may choose to edit and check the simulcast script
+rather than translate it from scratch. This depends almost
 entirely on the quality of the simulcast. Fixing a bad simulcast script
 may be harder than doing an *original translation* (abbreviated OTL).
 Finally, translators are responsible for transcribing and translating
 opening, ending, and insert songs as well.
 
-### Karaoke Effect Creator
+## Karaoke Effect Creator
 
 Time commitment: several hours, once or twice per season
 
-The Karaoke Effect Creator (always referred to as the *KFX*) styles and
+The Karaoke Effect Creator (abbreviated *KFX*) styles and
 adds effects to the lyrics and sometimes romaji and/or kanji for
 opening, ending, and insert songs. This can be very similar to
 typesetting but utilizes a different set of tools and can be highly
 programming-oriented.
+
+***
+
+<b name="fn1">1</b>: <kbd>TODO - sources for AviSynth and VapourSynth builds
```

Comment:
These should be covered in one of the encoding pages, iirc, but probably doesn't hurt to mention this here as well.